### ISSUE TITLE
[Feature]: Delete the database if I forgot my password

### ISSUE THREAD

**Issue by ElonSatoshi** [simplex-chat/simplex-chat#4492]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

all

### App version

5.8.2

### Feature

![image](https://github.com/user-attachments/assets/630c613a-cd93-4474-8a15-9d6836b9b49b)

If you are unfortunate enough to forget your password, you have lost access to your contacts and everything.

However, to add insult to injury, before you can start from scratch and build a new set of profiles while you begin to accept your loss, you have to figure out how to manually reset SimpleX Chat.

This involves locating where the config files and the database are stored (this varies greatly depending on the platform and the installation method), then deleting them. On Android, you have to know how get to the app info page on SimpleX Chat, then clear storage, to completely reset the app.

SimpleX should have a "Forgot password?" or "Delete account" button showing when it asks for your passphrase.

Briar has this feature, for reference. It warns you about what it does and asks for confirmation.

![image](https://github.com/user-attachments/assets/b3699541-8d1f-401c-bdf1-d1c54376544b)
![image](https://github.com/user-attachments/assets/1086b913-a38c-42bb-8787-388d1b732ce4)

**Comment by greensheeps** [comment]
> Let’s be honest, a secure system is when the app doesn’t help you find your password.
> 
> Otherwise you introduced a backdoor.
> 
> If you lose your password you have lost your account.
> 
> It is perfect. It must be so. It is a guarantee of security.

This isn't what they were asking. They were asking for a button to reset the database like Briar, not a usual forgot my password (backdoor) button

**Comment by amlkyc** [comment]
Where is the database file is stored to manually delete it?

**Comment by ElonSatoshi** [comment]
> Where is the database file stored to manually delete it? On Windows UPD: Deleted SimpleX folder from %APPDATA% and it worked

Glad you figured it out! Also, thanks for letting us know how you solved it for future reference, and thank you for illustrating the significance of this issue.

**Comment by kuligs2** [comment]
brugh, not being able to start a new db on the login screen is just bad design... i found this issue becasue i was facing the same problem. And its not that im losing everything, i never had nothing becasue xftp does not work on windows client. So there is no point in using the app in the first place if you cant sync between the devices.. 

And so im here because im trying to give this a second chance. Set up a new server, and now i could not start a new profile because the app does not provide such option.

**Comment by vasild** [comment]
https://simplex.chat/faq/#i-do-not-know-my-database-passphrase

**Comment by DeMoN100** [comment]
I agree that a delete db should be on the login screen that help page does not tell you exactly where it is on linux. It gives you a general idea but really when outlining where something is for linux you can't make the general home/wherever/etc you need to label it all put specifically for each distro. For example those instructions won't help me find it on Fedora maybe system 76 or ubuntu. It's a pain because theres so many distros but the one who does it looks better presented.

Bottom line the delete database button should be under login. Make it happen!

**Comment by menin99** [comment]
The fact that it has taken over a year and there has still been no movement on this is why nobody uses open source software. Half the replies are "no you shouldn't have this feature because I don't like it". Ridiculous

**Comment by ElonSatoshi** [comment]
Agreed. @epoberezkin please confirm whether this will be worked on, or if it is not planned.

**Comment by kuligs2** [comment]
> Agreed. [@epoberezkin](https://github.com/epoberezkin) please confirm whether this will be worked on, or if it is not planned.

Back to matrix bois, back to matrix...
